src/main/python/interfaces/widgets/module.py

```diff
@@ -227,10 +343,84 @@ def reset(self):
         super().reset()
 
 
+class AttendanceBox(Frame):
+    layout: HorizontalLayout = None
+    button: Button = None
+    frame: Frame = None
+    flag: bool = False
+
+    def __init__(self, widget: QWidget, *args, height: int = 38, **kwargs):
+
+        # lambda func grab if given
```

Comment:
What's up with this?

Reply:
My bad, I forget to add comment about this. It would just a wrapper for each attendance option. (i.e. rent immediately, rent at xx~xx).